Auto update failing for vulkan-tools

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the vulkan-tools package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating vulkan-tools [Current version: 1.3.242]
INFO: package being updated to 1.3.243.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 40979    0 40979    0     0  60118      0 --:--:-- --:--:-- --:--:-- 60118
100  760k    0  760k    0     0   784k      0 --:--:-- --:--:-- --:--:-- 2508k

You are about to commit these changes:

--------------------
diff --git a/packages/vulkan-tools/build.sh b/packages/vulkan-tools/build.sh
index 9707c6f63..295ff37fc 100644
--- a/packages/vulkan-tools/build.sh
+++ b/packages/vulkan-tools/build.sh
@@ -3,10 +3,9 @@ TERMUX_PKG_DESCRIPTION="Vulkan Tools and Utilities"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 # This package and vulkan-headers should be updated at same time. Otherwise, they do not compile successfully.
-TERMUX_PKG_VERSION="1.3.242"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="1.3.243"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-Tools/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=a1548442e5ec1440bce219136c100e8505e8d1f4c24666cab9f26abb953a3725
+TERMUX_PKG_SHA256=5037d7b09593d1b96e178c9a50cf233cb7671b9fc21b681b5de0bb17fde3392d
 TERMUX_PKG_DEPENDS="libc++, libx11, libxcb, libwayland, vulkan-loader"
 TERMUX_PKG_BUILD_DEPENDS="vulkan-headers (=${TERMUX_PKG_VERSION}), libwayland-protocols"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
--------------------

bump(main/vulkan-tools): 1.3.243


Not committing to Git!

INFO: Trying to build package.
INFO: Package vulkan-tools exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building vulkan-tools for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0   117k      0 --:--:-- --:--:-- --:--:--  117k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1430      0 --:--:-- --:--:-- --:--:--  1428
100   833  100   833    0     0   1430      0 --:--:-- --:--:-- --:--:--  1428
gpg: Signature made Tue 14 Mar 2023 12:34:36 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0  22447      0 --:--:-- --:--:-- --:--:-- 22460
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1341      0 --:--:-- --:--:-- --:--:--  1341
gpg: Signature made Thu 09 Mar 2023 04:56:31 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 79 13120   79 10413    0     0  17336      0 --:--:-- --:--:-- --:--:-- 17326
100 13120  100 13120    0     0  21832      0 --:--:-- --:--:-- --:--:-- 21830
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1375      0 --:--:-- --:--:-- --:--:--  1376
gpg: Signature made Mon 13 Mar 2023 11:34:12 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@25c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libc++/libc++_25c_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  216k  100  216k    0     0  1567k      0 --:--:-- --:--:-- --:--:-- 1577k
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@28-3 if necessary...
Skipping already built dependency libandroid-support@28-3
Downloading dependency libexpat@2.5.0 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libe/libexpat/libexpat_2.5.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 81732  100 81732    0     0   653k      0 --:--:-- --:--:-- --:--:--  654k
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.4 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libf/libffi/libffi_3.4.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 30692  100 30692    0     0   250k      0 --:--:-- --:--:-- --:--:--  251k
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.17 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libi/libiconv/libiconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 21  543k   21  115k    0     0   908k      0 --:--:-- --:--:-- --:--:--  904k
100  543k  100  543k    0     0  3752k      0 --:--:-- --:--:-- --:--:-- 3747k
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.17 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/i/iconv/iconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8988  100  8988    0     0  81686      0 --:--:-- --:--:-- --:--:-- 82458
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.4.1 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libl/liblzma/liblzma_5.4.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  173k  100  173k    0     0  1276k      0 --:--:-- --:--:-- --:--:-- 1284k
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.4.1 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xz-utils/xz-utils_5.4.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 63356  100 63356    0     0   502k      0 --:--:-- --:--:-- --:--:--  503k
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.4-1 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses/ncurses_6.4-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  502k  100  502k    0     0  3319k      0 --:--:-- --:--:-- --:--:-- 3328k
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.4-1 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-utils/ncurses-utils_6.4-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 15 85744   15 13145    0     0  14212      0  0:00:06 --:--:--  0:00:06 14210
100 85744  100 85744    0     0  91717      0 --:--:-- --:--:-- --:--:-- 91704
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.4-1 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs/ncurses-ui-libs_6.4-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 32452  100 32452    0     0   271k      0 --:--:-- --:--:-- --:--:--  273k
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.4-1 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs-static/ncurses-ui-libs-static_6.4-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 38932  100 38932    0     0   324k      0 --:--:-- --:--:-- --:--:--  324k
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.1 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/r/readline/readline_8.2.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  227k  100  227k    0     0  1661k      0 --:--:-- --:--:-- --:--:-- 1664k
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.23 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gdbm/gdbm_1.23_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  138k  100  138k    0     0  1094k      0 --:--:-- --:--:-- --:--:-- 1100k
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-posix-semaphore/libandroid-posix-semaphore_0.1-3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  4128  100  4128    0     0  36806      0 --:--:-- --:--:-- --:--:-- 36530
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libb/libbz2/libbz2_1.0.8-6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 26634  100 26634    0     0  35041      0 --:--:-- --:--:-- --:--:-- 35044
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/b/bzip2/bzip2_1.0.8-6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 25506  100 25506    0     0   215k      0 --:--:-- --:--:-- --:--:--  214k
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2023.01.10 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/ca-certificates/ca-certificates_1:2023.01.10_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  115k  100  115k    0     0   894k      0 --:--:-- --:--:-- --:--:--  898k
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2023.01.10 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/ca-certificates-java/ca-certificates-java_1:2023.01.10_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  105k  100  105k    0     0   826k      0 --:--:-- --:--:-- --:--:--  828k
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.2.13 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/z/zlib/zlib_1.2.13_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 60060  100 60060    0     0   433k      0 --:--:-- --:--:-- --:--:--  434k
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.0.8 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/openssl/openssl_1:3.0.8_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 96 1540k   96 1486k    0     0  7024k      0 --:--:-- --:--:-- --:--:-- 7011k
100 1540k  100 1540k    0     0  7256k      0 --:--:-- --:--:-- --:--:-- 7230k
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.0.8 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/openssl-tool/openssl-tool_1:3.0.8_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  234k  100  234k    0     0  1749k      0 --:--:-- --:--:-- --:--:-- 1762k
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libcrypt/libcrypt_0.2-5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8720  100  8720    0     0  73329      0 --:--:-- --:--:-- --:--:-- 73898
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.41.1 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libs/libsqlite/libsqlite_3.41.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  561k  100  561k    0     0  3340k      0 --:--:-- --:--:-- --:--:-- 3360k
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.41.1 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/sqlite/sqlite_3.41.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  527k  100  527k    0     0  3602k      0 --:--:-- --:--:-- --:--:-- 3612k
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.13 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/tcl/tcl_8.6.13_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0 2191k    0  9044    0     0   9487      0  0:03:56 --:--:--  0:03:56  9480
 36 2191k   36  797k    0     0   496k      0  0:00:04  0:00:01  0:00:03  496k
100 2191k  100 2191k    0     0  1231k      0  0:00:01  0:00:01 --:--:-- 1231k
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.0.9-1 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/b/brotli/brotli_1.0.9-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  296k  100  296k    0     0  2159k      0 --:--:-- --:--:-- --:--:-- 2165k
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.39 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libp/libpng/libpng_1.6.39_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  191k  100  191k    0     0  1339k      0 --:--:-- --:--:-- --:--:-- 1336k
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.0-1 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/freetype/freetype_2.13.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  397k  100  397k    0     0  2725k      0 --:--:-- --:--:-- --:--:-- 2739k
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/ttf-dejavu/ttf-dejavu_2.37-8_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2410k  100 2410k    0     0  13.1M      0 --:--:-- --:--:-- --:--:-- 13.1M
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.14.2-1 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/fontconfig/fontconfig_2.14.2-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  115k  100  115k    0     0   921k      0 --:--:-- --:--:-- --:--:--  923k
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.14.2-1 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/fontconfig-utils/fontconfig-utils_2.14.2-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 14336  100 14336    0     0   124k      0 --:--:-- --:--:-- --:--:--  125k
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.0 if necessary...
Skipping already built dependency xorg-util-macros@1.20.0
Downloading dependency xorgproto@2022.2 if necessary...
Skipping already built dependency xorgproto@2022.2
Downloading dependency libxau@1.0.11 if necessary...
Skipping already built dependency libxau@1.0.11
Downloading dependency libxdmcp@1.1.4 if necessary...
Skipping already built dependency libxdmcp@1.1.4
Downloading dependency xcb-proto@1.15.2-4 if necessary...
Skipping already built dependency xcb-proto@1.15.2-4
Downloading dependency python-xcbgen@1.15.2-4 if necessary...
Skipping already built dependency python-xcbgen@1.15.2-4
Downloading dependency libxcb@1.15 if necessary...
Skipping already built dependency libxcb@1.15
Downloading dependency xtrans@1.4.0-9 if necessary...
Skipping already built dependency xtrans@1.4.0-9
Downloading dependency libx11@1.8.4-1 if necessary...
Skipping already built dependency libx11@1.8.4-1
Downloading dependency libxrender@0.9.11 if necessary...
Skipping already built dependency libxrender@0.9.11
Downloading dependency libxft@2.3.7 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxft/libxft_2.3.7_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 47808  100 47808    0     0   379k      0 --:--:-- --:--:-- --:--:--  382k
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.5 if necessary...
Skipping already built dependency libxext@1.3.5
Downloading dependency libxss@1.2.3-13 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxss/libxss_1.2.3-13_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 11666  100 11666    0     0   102k      0 --:--:-- --:--:-- --:--:--  103k
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.13 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/t/tk/tk_8.6.13_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1701k  100 1701k    0     0  9868k      0 --:--:-- --:--:-- --:--:-- 9893k
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.11.2-2 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python/python_3.11.2-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 7811k  100 7811k    0     0  38.3M      0 --:--:-- --:--:-- --:--:-- 38.5M
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.11.2-2 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-tkinter/python-tkinter_3.11.2-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1193k  100 1193k    0     0  6695k      0 --:--:-- --:--:-- --:--:-- 6706k
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.11.2-2 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/python-ensurepip-wheels/python-ensurepip-wheels_3.11.2-2_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2854k  100 2854k    0     0  13.9M      0 --:--:-- --:--:-- --:--:-- 13.9M
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency libxml2@2.10.3-1 if necessary...
Found libxml2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2/libxml2_2.10.3-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  520k  100  520k    0     0  3546k      0 --:--:-- --:--:-- --:--:-- 3566k
extracting libxml2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-utils@2.10.3-1 if necessary...
Found libxml2-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-utils/libxml2-utils_2.10.3-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 30672  100 30672    0     0   269k      0 --:--:-- --:--:-- --:--:--  272k
extracting libxml2-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python@2.10.3-1 if necessary...
Found libxml2-python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-python/libxml2-python_2.10.3-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  199k  100  199k    0     0   318k      0 --:--:-- --:--:-- --:--:--  319k
extracting libxml2-python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python-static@2.10.3-1 if necessary...
Found libxml2-python-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-python-static/libxml2-python-static_2.10.3-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 69044  100 69044    0     0   556k      0 --:--:-- --:--:-- --:--:--  557k
extracting libxml2-python-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland@1.21.0-1 if necessary...
Found libwayland in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland/libwayland_1.21.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 35  113k   35 41620    0     0  37700      0  0:00:03  0:00:01  0:00:02 37699
100  113k  100  113k    0     0   101k      0  0:00:01  0:00:01 --:--:--  101k
extracting libwayland to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland-cross-scanner@1.21.0-1 if necessary...
Found libwayland-cross-scanner in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland-cross-scanner/libwayland-cross-scanner_1.21.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 80416  100 80416    0     0   126k      0 --:--:-- --:--:-- --:--:--  126k
extracting libwayland-cross-scanner to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland-protocols@1.31-1 if necessary...
Found libwayland-protocols in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland-protocols/libwayland-protocols_1.31-1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 72884  100 72884    0     0  79139      0 --:--:-- --:--:-- --:--:-- 79135
extracting libwayland-protocols to /home/builder/.termux-build/_cache-all...
Downloading dependency vulkan-headers@1.3.243 if necessary...
Skipping already built dependency vulkan-headers@1.3.243
Downloading dependency vulkan-loader-android@25c if necessary...
Found vulkan-loader-android in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-loader-android/vulkan-loader-android_25c_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  5160  100  5160    0     0   8642      0 --:--:-- --:--:-- --:--:--  8643
extracting vulkan-loader-android to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrandr@1.5.3 if necessary...
Skipping already built dependency libxrandr@1.5.3
Downloading dependency vulkan-loader-generic@1.3.243 if necessary...
Skipping already built dependency vulkan-loader-generic@1.3.243
Downloading dependency vulkan-loader@0.0.1 if necessary...
Found vulkan-loader in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-loader/vulkan-loader_0.0.1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   736  100   736    0     0   1229      0 --:--:-- --:--:-- --:--:--  1228
100   736  100   736    0     0   1229      0 --:--:-- --:--:-- --:--:--  1228
extracting vulkan-loader to /home/builder/.termux-build/_cache-all...
Downloading https://github.com/KhronosGroup/Vulkan-Tools/archive/v1.3.243.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  442k    0  442k    0     0   542k      0 --:--:-- --:--:-- --:--:--  542k
100  760k    0  760k    0     0   823k      0 --:--:-- --:--:-- --:--:-- 2993k
Applying patch: no-android.patch
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The C compiler identification is Clang 14.0.7
-- The CXX compiler identification is Clang 14.0.7
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/builder/.termux-build/_cache/android-r25c-api-24-v1/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r25c-api-24-v1/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Using find_package to locate Vulkan
-- Found Vulkan: /data/data/com.termux/files/usr/lib/libvulkan.so  
-- Vulkan FOUND = TRUE
-- Vulkan Lib Dir = /data/data/com.termux/files/usr/lib
-- Vulkan Lib = /data/data/com.termux/files/usr/lib/libvulkan.so
-- Using cmake find_program to look for glslangValidator
   Downloading glslangValidator binary from glslang releases dir
-- Found PkgConfig: /home/builder/.termux-build/_cache/android-r25c-api-24-v1/bin/pkg-config (found version "0.29.2") 
-- Found WAYLAND_CLIENT: /data/data/com.termux/files/usr/lib/libwayland-client.so  
-- Found WAYLAND_SERVER: /data/data/com.termux/files/usr/lib/libwayland-server.so  
-- Found WAYLAND_EGL: /data/data/com.termux/files/usr/lib/libwayland-egl.so  
-- Found WAYLAND_CURSOR: /data/data/com.termux/files/usr/lib/libwayland-cursor.so  
-- Found WAYLAND: /data/data/com.termux/files/usr/lib/libwayland-client.so;/data/data/com.termux/files/usr/lib/libwayland-server.so;/data/data/com.termux/files/usr/lib/libwayland-egl.so;/data/data/com.termux/files/usr/lib/libwayland-cursor.so  
-- Found WAYLAND: /data/data/com.termux/files/usr/opt/libwayland/cross/bin/wayland-scanner  
-- Checking for module 'wayland-protocols'
--   Found wayland-protocols, version 1.31
-- Found WAYLAND: //data/data/com.termux/files/usr/share/wayland-protocols  
-- Looking for clock_gettime in rt
-- Looking for clock_gettime in rt - found
-- Found WAYLAND: /data/data/com.termux/files/usr/lib/libwayland-client.so;/data/data/com.termux/files/usr/lib/libwayland-server.so;/data/data/com.termux/files/usr/lib/libwayland-egl.so;/data/data/com.termux/files/usr/lib/libwayland-cursor.so  
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


-- Build files have been written to: /home/builder/.termux-build/vulkan-tools/build
[1/16] Compiling cube vertex shader
/home/builder/.termux-build/vulkan-tools/src/cube/cube.vert
[2/16] Compiling cube fragment shader
/home/builder/.termux-build/vulkan-tools/src/cube/cube.frag
[3/16] Generating xdg-shell protocol dispatch data
[4/16] Building C object cube/CMakeFiles/vkcube.dir/cube.c.o
[5/16] Linking C executable cube/vkcube
FAILED: cube/vkcube 
: && /home/builder/.termux-build/_cache/android-r25c-api-24-v1/bin/clang --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r25c-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r25c-api-24-v1/sysroot -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -std=c99 -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -fno-strict-aliasing -fno-builtin-memcmp -Werror -fvisibility=hidden -O3 -DNDEBUG -Wl,--gc-sections -L/data/data/com.termux/files/usr/lib -Wl,-rpath=/data/data/com.termux/files/usr/lib -fopenmp -static-openmp -Wl,--enable-new-dtags -Wl,--as-needed -Wl,-z,relro,-z,now --target=aarch64-linux-android24    -fPIE -pie cube/CMakeFiles/vkcube.dir/cube.c.o -o cube/vkcube  -lm  /data/data/com.termux/files/usr/lib/libvulkan.so  -lrt  /data/data/com.termux/files/usr/lib/libxcb.so && :
ld: error: undefined symbol: vkCreateXcbSurfaceKHR
>>> referenced by cube.c
>>>               cube/CMakeFiles/vkcube.dir/cube.c.o:(main)
>>> referenced by cube.c
>>>               cube/CMakeFiles/vkcube.dir/cube.c.o:(main)
>>> referenced by cube.c
>>>               cube/CMakeFiles/vkcube.dir/cube.c.o:(main)
clang-14: error: linker command failed with exit code 1 (use -v to see invocation)
[6/16] Building CXX object cube/CMakeFiles/vkcubepp.dir/cube.cpp.o
ninja: build stopped: subcommand failed.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2023-03-14 00:41:37 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/4410791478">4410791478</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/vulkan-tools/build.sh
+++ b/packages/vulkan-tools/build.sh
@@ -3,12 +3,11 @@ TERMUX_PKG_DESCRIPTION="Vulkan Tools and Utilities"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 # This package and vulkan-headers should be updated at same time. Otherwise, they do not compile successfully.
-TERMUX_PKG_VERSION="1.3.242"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="1.3.243"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-Tools/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=a1548442e5ec1440bce219136c100e8505e8d1f4c24666cab9f26abb953a3725
+TERMUX_PKG_SHA256=5037d7b09593d1b96e178c9a50cf233cb7671b9fc21b681b5de0bb17fde3392d
 TERMUX_PKG_DEPENDS="libc++, libx11, libxcb, libwayland, vulkan-loader"
-TERMUX_PKG_BUILD_DEPENDS="vulkan-headers (=${TERMUX_PKG_VERSION}), libwayland-protocols"
+TERMUX_PKG_BUILD_DEPENDS="libwayland-protocols, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-loader-generic (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 -DBUILD_CUBE=ON
 -DBUILD_ICD=OFF

